platform/ios/scripts/package.sh

```diff
@@ -29,7 +28,7 @@ SDK=iphonesimulator
 if [[ ${BUILD_FOR_DEVICE} == true ]]; then
     SDK=iphoneos
 fi
-IOS_SDK_VERSION=`xcrun --sdk ${SDK} --show-sdk-version`
+IOS_SDK_VERSION='v4.5.0-cn.1'
```

Comment:
This shouldn’t be changed. `IOS_SDK_VERSION` is purely informational  — it’s meant to show what version of the [iOS (system) SDK](https://en.wikipedia.org/wiki/IOS_SDK) _our_ maps SDK is being built against.